core/go/internal/plugins/domains.go

```diff
@@ -221,13 +222,17 @@ func (br *domainBridge) InitTransaction(ctx context.Context, req *prototk.InitTr
 }
 
 func (br *domainBridge) AssembleTransaction(ctx context.Context, req *prototk.AssembleTransactionRequest) (res *prototk.AssembleTransactionResponse, err error) {
+	log.L(ctx).Infof("[Plugin] assembling transaction %s", req.Transaction.TransactionId)
 	err = br.toPlugin.RequestReply(ctx,
 		func(dm plugintk.PluginMessage[prototk.DomainMessage]) {
 			dm.Message().RequestToDomain = &prototk.DomainMessage_AssembleTransaction{AssembleTransaction: req}
 		},
 		func(dm plugintk.PluginMessage[prototk.DomainMessage]) bool {
 			if r, ok := dm.Message().ResponseFromDomain.(*prototk.DomainMessage_AssembleTransactionRes); ok {
 				res = r.AssembleTransactionRes
+				for i, attRequest := range res.AttestationPlan {
+					log.L(ctx).Infof("[Plugin] assembled transaction %s, assemble response %d name '%s'", req.Transaction.TransactionId, i, attRequest.Name)
```

Comment:
Why info level for just this one request to a plugin, and why this info?
We have very detailed debug logging to plugins. If there's a good reason for escalated info level logging that's something be be considered carefully I think.